canonical-web-and-design/kubeflow-news.com
# 1269 pass blog data to templates

## Done

- added blog module
- created a tag on insights.ubuntu.com for "Kubeflow News" (id: 3408)
- setup basic templates for:
  - index page (to feature only the latest article)
  - individual articles
  - archives

## QA

- Check out this branch
- Run `./run clean && ./run`
- In your text editor, on line 14 of webapp/app.py, replace:
  - `[3408]` with `[2389]`
  - `'kubeflow-news'` with `'ai'` (because no posts yet have the kubeflow news tag)
- Visit [http://0.0.0.0:8035](http://0.0.0.0:8035), and see that there is a heading and content from the most recent article tagged with 'AI'
- Visit [/archives](http://0.0.0.0:8035/feed), and see that there is a list of articles tagged with 'AI'
- Visit [/feed](http://0.0.0.0:8035/feed), and see that there is an XML list of articles
